vendor/github.com/docker/libnetwork/drivers/windows/windows.go

```diff
@@ -646,7 +646,7 @@ func (d *driver) CreateEndpoint(nid, eid string, ifInfo driverapi.InterfaceInfo,
 	}
 
 	if err = d.storeUpdate(endpoint); err != nil {
-		return fmt.Errorf("failed to save endpoint %s to store: %v", endpoint.id[0:7], err)
+		logrus.Errorf("Failed to save endpoint %s to store: %v", endpoint.id[0:7], err)
```

Comment:
This may warrant a comment, explaining that it's an deliberate choice to only printed/log the error, and not handle it as an error

Reply:
@msabansal something to consider as a followup in libnetwork

Reply:
I have plans to remove storage completely in next release with the introduction of labels in HNS. With that we will allow service to know who owns which data and they can clean their own data.